File: People.cpp
// People.cpp
#include "People.hpp"

using namespace InfSystBusStation;

void People::PrintInfo() {
    Console::WriteLine("=== Информация о человеке ===");
    Console::WriteLine("ФИО: {0}", fullName);
    Console::WriteLine("Пол: {0}", gender);
    if (!String::IsNullOrEmpty(passportSeries) && !String::IsNullOrEmpty(passportNumber)) {
        Console::WriteLine("Паспорт: {0} {1}", passportSeries, passportNumber);
    }
    if (!String::IsNullOrEmpty(email)) {
        Console::WriteLine("Email: {0}", email);
    }
}

String^ People::GetFullInfo() {
    String^ info = fullName;

    if (!String::IsNullOrEmpty(passportSeries) && !String::IsNullOrEmpty(passportNumber)) {
        info += " (" + passportSeries + " " + passportNumber + ")";
    }

    return info;
}

File: People.java
public class People {
    protected String surname;
    protected String name;
    protected String middleName;
    protected String passportSeries;
    protected String passportNumber;
    protected String email;

    // Конструктор
    public People(String surname, String name, String middleName, 
                 String passportSeries, String passportNumber, String email) {
        this.surname = surname;
        this.name = name;
        this.middleName = middleName;
        this.passportSeries = passportSeries;
        this.passportNumber = passportNumber;
        this.email = email;
    }

    // Публичные методы класса
    public void printInfo() {
        System.out.println("[People] Full name: " + surname + " " + name + " " + middleName
                + "\nPassport: " + passportSeries + " " + passportNumber
                + "\nEmail: " + email + "\n");
    }

    public String getFullName() {
        return surname + " " + name + " " + middleName;
    }

    // Геттеры
    public String getSurname() {
        return surname;
    }

    public String getName() {
        return name;
    }

    public String getMiddleName() {
        return middleName;
    }

    public String getPassportSeries() {
        return passportSeries;
    }

    public String getPassportNumber() {
        return passportNumber;
    }

    public String getEmail() {
        return email;
    }

    // Сеттеры
    public void setSurname(String surname) {
        this.surname = surname;
    }

    public void setName(String name) {
        this.name = name;
    }

    public void setMiddleName(String middleName) {
        this.middleName = middleName;
    }

    public void setPassportSeries(String passportSeries) {
        this.passportSeries = passportSeries;
    }

    public void setPassportNumber(String passportNumber) {
        this.passportNumber = passportNumber;
    }

    public void setEmail(String email) {
        this.email = email;
    }
}